doc/policies/webui.rst

```diff
@@ -334,6 +334,19 @@ being disabled.
 
 (Since privacyIDEA 3.0)
 
+deletion_confirmation
+~~~~~~~~~~~~~~~~~~~~~
+
+type: ``bool``
+
+To avoid careless deletion of important configurations, this policy can be
+activated. After activation, an additional confirmation for the deletion is
+requested for policies, events, mresolvers, resolvers and periodic-tasks.
+
+.. note:: After activating the policy, the UI must be reloaded once for the change to take effect
```

Comment:
I think we should remove this note and add a general note for all webUI policies at the beginning of the text.

Reply:
Done in f85deca5